### ISSUE TITLE
[GH Request] Core Contributor access: Pooja

### ISSUE THREAD

**Issue by pkulkark** [openedx/tcril-engineering#292]
### Firm Name

tCRIL

### Urgency

Medium

### Requested Change

Pooja has been approved as a Core Contributor to: edx-platform and completion. Please grant her write access to these repos.

### Reasoning

Approved via CC process nominated by @pomegranited : https://discuss.openedx.org/t/core-contributor-nomination-pooja-kulkarni/7265

**Comment by sarina** [comment]
Hi @openedx/tcril-oncall : this ticket is meant to be filed after the legal documents are signed.

@pkulkark when you get the confirmation that this has been countersigned by tCRIL Legal, could you reopen this ticket to say it's ready to go?

**Comment by pkulkark** [comment]
@sarina Oops! I thought this could be done in parallel. Will reopen once the legal documents are countersigned. Sorry for the noise.

**Comment by pkulkark** [comment]
Thanks @sarina, it's much better now. The legal documents are countersigned but I don't seem to have access to reopen this. Do I need to create a new issue? CC @arbrandes

**Comment by arbrandes** [comment]
@pkulkark, reopened!  I'm adding this to my TODO list for this week.  (Congrats, by the way!)

**Comment by pkulkark** [comment]
Thank you @arbrandes!

**Comment by sarina** [comment]
@arbrandes I added Pooja to the Slack group, and Dave added her to the Google groups. Let me know if there's anything else I can help with.

**Comment by arbrandes** [comment]
Thanks @sarina!  This is now done.
